lib/sinatra/show_exceptions.rb

```diff
@@ -24,23 +24,22 @@ def call(env)
 
       if prefers_plain_text?(env)
         content_type = "text/plain"
-        exception_string = dump_exception(e)
+        exception = dump_exception(e)
       else
         content_type = "text/html"
-        exception_string = pretty(env, e)
+        exception = pretty(env, e)
       end
 
       env["rack.errors"] = errors
-
-      # Post 893a2c50 in rack/rack, the #pretty method above, implemented in
-      # Rack::ShowExceptions, returns a String instead of an array.
-      body = Array(exception_string)
+      body_array = exception.is_a?(String) ? [exception] : exception
```

Comment:
Yeah it was intended to make explicit that the return could be a string, just looking to the code without a comment explaining it.

But for sure, the Array declaration before was simpler (and maybe faster).

Before this change I tried to see how the implementation of the Array method works (in the ruby's kernel), to see if it uses an if too, but I didn't find the part of the code that does it.


So if we continue using the Array expression, we will need the comment again because it will be implicit that the return could also be a string.

What do you think @kytrinyx @kgrz?